### PR TITLE
Update hydra editor in 2.x stable

### PR DESCRIPTION
**Pull request by geekscruff**
Upgrade hydra-editor and simple_form, and fix failing tests caused by changes to simple_form.

### CODE REVIEW

File: hyrax.gemspec [hunk 1]
```diff
@@ -49,7 +49,7 @@ SUMMARY
   spec.add_dependency 'flot-rails', '~> 0.0.6'
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
-  spec.add_dependency 'hydra-editor', '>= 3.3', '< 5.0'
+  spec.add_dependency 'hydra-editor', '~> 5.0'
```
Comment (no-reply) [inline hyrax.gemspec]: is it possible to do this without dropping `hydra-editor` 3.3 and 4.x support?

Comment (geekscruff) [reply hyrax.gemspec]: I've pushed an update ... would the removal of the to_model method in `app/forms/hyrax/forms/workflow_responsibility_form.rb` be a problem?

Comment (geekscruff) [reply hyrax.gemspec]: well the tests still pass

File: hyrax.gemspec [hunk 2]
```diff
@@ -49,7 +49,7 @@ SUMMARY
   spec.add_dependency 'flot-rails', '~> 0.0.6'
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
-  spec.add_dependency 'hydra-editor', '>= 3.3', '< 5.0'
+  spec.add_dependency 'hydra-editor', '>= 3.0', '~> 5.0'
```
Comment (cjcolvar) [inline hyrax.gemspec]: Does this allow a 3.x hydra-editor?  I would think that the `~> 5.0` would force to only 5.x releases.  Maybe `< 6` would work?

Comment (geekscruff) [reply hyrax.gemspec]: The idea was to allow for 3.x yes so I've made that change. Do you think the change that removes to_model from the WorkflowResponsiblity would be problematic?

Comment (geekscruff) [reply hyrax.gemspec]: Ideally I'd like to force the upgrade on people ;)

Comment (geekscruff) [reply hyrax.gemspec]: Tests still passing with that change @cjcolvar

**Pull request merged**